frontend/src/views/fetch.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+import { useLazyQuery } from "@apollo/react-hooks";
+import gql from "graphql-tag";
+import { Input, Button } from "semantic-ui-react";
+import cogoToast from "cogo-toast";
+
+const FETCH_USER = gql`
+  query user($id: ID!) {
+    user(id: $id) {
+      email
+      id
+      lastName
+      firstName
+      hashedPwd
+    }
+  }
+`;
+
+export const Fetch = () => {
+  const [id, setId] = useState(null);
+
+  const [fetchUser, { data }] = useLazyQuery(FETCH_USER);
+
+  useEffect(() => {
+    console.log("data", data);
+  }, [data]);
+
+  return (
+    <div>
+      <div>
+        <h3>fetch user</h3>
+        <Input onChange={(e) => setId(e.target.value)} />
```

Comment:
takie funkcje lepiej wyciagac poza rendera zeby funkcja sie nie tworzyla za kazdym razem czyli onChange={handleInputChange}

i to handleInputChange np jako funciton pod returnem